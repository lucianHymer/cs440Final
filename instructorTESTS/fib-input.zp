//  File: fib-input.zp
//  Expected output: Fibonacci numbers up to n

int main () 
{
  int n;
//  n = getnum();
    n = 10;

  puts("Fibonacci numbers\n");
  puts("Up to ");
  putn(n);
  puts("\n");

  int lo; 
  int hi;
  lo = 1;
  hi = lo;
  putn(lo);
  putn(hi);
  puts("+++++++++++\n");

  while (hi < n) {
    putn(hi);
    hi = lo + hi;
    lo = hi - lo;
  }
  puts("+++++++++++\n");
  return 0;
}
