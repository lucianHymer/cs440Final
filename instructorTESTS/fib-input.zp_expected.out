Fibonacci numbers
Up to     10 

     1 
     1 
+++++++++++
     1 
     2 
     3 
     5 
     8 
+++++++++++
